api/client/proto/authservice.proto

```diff
@@ -1348,4 +1363,17 @@ service AuthService {
     // any existing codes the user previously had.
     rpc ChangeUserAuthentication(ChangeUserAuthenticationRequest)
         returns (ChangeUserAuthenticationResponse);
+
+    // StartAccountRecovery exclusive to cloud users, is the first out of two step user verification
```

Comment:
nit: Sentence seems a bit off? This may be a fix, although it's not the best either.

```suggestion
    // StartAccountRecovery, exclusive to cloud users, is the first out of two step user verification
```

Reply:
I placed a paranthesis: `// StartAccountRecovery (exclusive to cloud users) `